core/src/main/java/io/aiven/klaw/service/CommonUtilsService.java

```diff
@@ -1,7 +1,6 @@
 package io.aiven.klaw.service;
 
 import static io.aiven.klaw.helpers.KwConstants.ORDER_OF_TOPIC_ENVS;
-import static io.aiven.klaw.helpers.KwConstants.REQUEST_SCHEMA_OF_ENVS;
```

Comment:
we can also delete this constant declaration in KwConstants

Reply:
done!